### ISSUE TITLE
Support common protos for shopping  

### ISSUE THREAD

**Issue by blakeli0** [googleapis/sdk-platform-java#2018]
We don't have a solution for `non-cloud org level shared protos` currently, see [TypesProto](https://github.com/googleapis/google-cloud-java/pull/9811/files) for merchant/reports API and [TypesProto](https://github.com/googleapis/google-cloud-java/pull/9810/files#diff-56c7e76a27fff5470ae0a8c4ad162dc38086af4126834fb1a4e3211a5c7c5820) for merchant/inventories API that generated duplicated classes for the same shared protos.
In short term, we would need to manually create a new module for these common protos for shopping org and set up automations to automatically update the module if there is any changes to the protos, or add them to `java-common-protos`. 
In long term, we need to have a defined process for these `non-cloud org level shared protos` in case it happens to other orgs.

**Comment by blakeli0** [comment]
Revert changes in https://github.com/googleapis/sdk-platform-java/pull/2414 once this issue is fixed.

**Comment by zhumin8** [comment]
We encountered similar issue again in new lib generation for chat:
https://github.com/googleapis/google-cloud-java/pull/10471#issuecomment-1979389057
[google.apps.card.v1](https://github.com/googleapis/googleapis/blob/master/google/apps/card/v1/card.proto) is needed as dependency for chat.
